product_docs/docs/tpa/23/architecture-PGD-Always-ON.mdx

```diff
@@ -80,6 +86,7 @@ data centre that provides a level of redundancy, in whatever way
 this definition makes sense to your use case. For example, AWS
 regions, your own data centres, or any other designation to identify
 where your servers are hosted.
+!!!
```

Comment:
I think we can get rid of this line. 
![Screenshot 2024-08-22 at 09 55 10](https://github.com/user-attachments/assets/e98a0dbf-e7d8-431f-a04b-f8affaee2e50)